### ISSUE TITLE
Send remaining balance when claiming tokens

### ISSUE THREAD

**Issue by Deams51** [PrimaBlock/presale-pool#3]
It could be interesting and cheap enough to send back the remaining balance is there's any during the transferMyTokens operation.

**Comment by tamirms** [comment]
@Deams51 you mean the remaining balance for the pool?

**Comment by Deams51** [comment]
@tamirms Not sure. 
Let me rephrase. When a contributor request his tokens, if he has any remaining ether left in the pool due to chances in settings etc, then we could send back his ether remaining at the same time as the tokens.

**Comment by tamirms** [comment]
very good suggestion, I will do that

**Comment by leftparen** [comment]
Do we want this to occur for withdrawing *all* tokens as well? Or just when an individual claims their tokens?
